File: libcommon/src/main/java/com/soaic/libcommon/recyclerview/holder/BasicItemHolder.java
package com.soaic.libcommon.recyclerview.holder;

import android.support.v7.widget.RecyclerView;
import android.view.View;

public class BasicItemHolder extends RecyclerView.ViewHolder {

    public BasicItemHolder(View itemView) {
        super(itemView);
    }

    public <T extends View> T findViewById(int viewId){
        return itemView.findViewById(viewId);
    }
}

File: libcommont/src/main/java/com/soaic/libcommont/recyclerview/holder/BasicItemHolder.kt
package com.soaic.libcommont.recyclerview.holder

import android.support.v7.widget.RecyclerView
import android.view.View

class BasicItemHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    fun <T: View> findViewById(viewId: Int):T{
        return itemView.findViewById(viewId)
    }

}